lldb/source/Target/VerboseTrapFrameRecognizer.cpp

```diff
@@ -9,13 +9,28 @@
 
 #include "lldb/Utility/LLDBLog.h"
 #include "lldb/Utility/Log.h"
+#include "lldb/Utility/RegularExpression.h"
 
 #include "clang/CodeGen/ModuleBuilder.h"
 
 using namespace llvm;
 using namespace lldb;
 using namespace lldb_private;
 
+#ifdef LLDB_ENABLE_SWIFT
+/// Returns true if \ref frame_name is the name of
+/// a Swift frame we wouldn't want to stop in.
+static bool ShouldHideSwiftFrame(llvm::StringRef frame_name) {
+  static llvm::SmallVector<RegularExpression, 2> s_patterns{
+      RegularExpression{R"(^(__C\.)?std\.)"},
+      RegularExpression{R"(protocol witness for Cxx\.)"}};
```

Comment:
Would it be theoretically possible to have the Swift hidden frame recognizer take over in this case? It seems wrong to maintain this list in two places.